### PR TITLE
1

### PR DESCRIPTION
**Pull request by MAIK-01**
[DEMO LINK](https://MAIK-01.github.io/react_add-todo-form/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,17 +1,86 @@
...
+              Choose a user
+            </option>
+            {users.map(user => (
+              <option value={user.id} key={user.id}>
+                {user.name}
+              </option>
+            ))}
+          </select>
+        </div>
+        <div className={inputError ? 'input__error' : ''}>
```
Comment (alexandra-protyanova) [inline src/App.tsx:66]: u can use classnames library

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]
Add rest of files, and completed to todo info

File: src/App.tsx [hunk 2]
```diff
@@ -1,17 +1,95 @@
...
 
 const App: React.FC = () => {
+  const [todos, setTodos] = useState(todosFromServer);
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState('0');
+  const [inputError, setInputError] = useState(false);
+  const [selectError, setSelectError] = useState(false);
+  const userInfo = users.find(user => user.id === +userId);
+
+  const handlerSubmit = (event: { preventDefault: () => void; }) => {
```
Comment (ArtemTopolskyi) [inline src/App.tsx:18]:
```suggestion
  const handlerSubmit = (event: { preventDefault: () => void; }) => {
```
use provided react type

File: src/components/TodoList/TodoList.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+import { Todo } from '../../Type/Todo';
+
+type Props = {
+  todos: Todo[];
+  userId: number;
+};
+
+export const TodoList: React.FC<Props> = ({ todos, userId }) => {
+  const vsbTodos = todos
```
Comment (ArtemTopolskyi) [inline src/components/TodoList/TodoList.tsx:11]: what means this variable name

**Review by ArtemTopolskyi — APPROVED** [review]

**Pull request closed without merging**